app/Services/TaskActivityService.php

```diff
@@ -53,7 +50,7 @@ public function sortCollection(array &$collection, string $sort): void
         });
     }
 
-    public function getPaginateCollection(array $collection, int $currentPage, $perPage = 10): LengthAwarePaginator
+    private function _getPaginateCollection(array $collection, int $currentPage, $perPage = 10): LengthAwarePaginator
```

Comment:
PHP EA: You shouldn't use underscore as an indicator of *private* function

Reply:
remove it class